From 1b7cd9d0f88188af96c5379b34f0f1a8ee779b44 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 27 Mar 2020 12:33:32 +0000
Subject: [PATCH 1014/1077] MANTIS 0056312 extension: Extend Desko-Penta
 Device-Handler by Accufill-SDK for reading US driving-licenses

fixed case of schema file name

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1565 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/idl/Cuss/{IdDocument.xsd => IdDocument.XSD}            | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename SelfServiceCommon/Massai/idl/Cuss/{IdDocument.xsd => IdDocument.XSD} (100%)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/IdDocument.xsd b/SelfServiceCommon/Massai/idl/Cuss/IdDocument.XSD
similarity index 100%
rename from SelfServiceCommon/Massai/idl/Cuss/IdDocument.xsd
rename to SelfServiceCommon/Massai/idl/Cuss/IdDocument.XSD
-- 
2.41.0.windows.1

